chart/templates/ws-proxy-deployment.yaml

```diff
@@ -34,6 +34,10 @@ spec:
         component: ws-proxy
         kind: pod
         stage: {{ .Values.installation.stage }}
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: '60095'
```

Comment:
On other deployments, I see `9500` instead of `60095`. Is this one an exception?

Reply:
Yes, because ws-proxy works with ports a lot. 9500 would likely collide with regular operations.